Rubrik/Public/Connect-Rubrik.ps1

```diff
@@ -25,6 +25,14 @@ function Connect-Rubrik
       The prompt will request a secure password.
 
       .EXAMPLE
+      Connect-Rubrik -Server 192.168.1.1 -Username admin -PasswordFile C:\pwd.txt
+      
+      To create the password file:
+      Read-Host -AsSecureString | ConvertFrom-SecureString | Out-File '.\pwd.txt'
```

Comment:
Can anyone use the password file? Seems like it would be encrypted to that user's SID.